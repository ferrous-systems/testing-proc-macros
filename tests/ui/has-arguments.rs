#[contracts::contracts(a, b)]
fn f() {}

fn main() {}
