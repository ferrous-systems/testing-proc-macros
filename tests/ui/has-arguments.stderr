error: this attribute takes no arguments

         = help: use `#[contracts]`

 --> $DIR/has-arguments.rs:3:1
  |
3 | #[contracts(a, b)]
  | ^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `contracts` (in Nightly builds, run with -Z macro-backtrace for more info)
